tests/PHPStan/Rules/Functions/data/unset.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+unset($notSetVariable);
+
+$scalar = 3;
+
+unset($scalar['a']);
+
+$singleDimArray = ['a' => 1];
+
+unset($singleDimArray['a']['b']);
+
+$multiDimArray = ['a' => ['b' => 1]];
+
+unset($multiDimArray['a']['b']['c'], $scalar, $singleDimArray['a']['b']);
+
+if (rand()) {
+	$maybeSet = 1;
+}
+
+unset($maybeSet);
+
+/** @param iterable<int> $iterable */
+function unsetOnMaybeIterable(iterable $iterable) {
```

Comment:
Don't mix side effects and function definitions. Please move all the surrounding test code from root scope to an anonymous function scope.